client/src/modules/stock/lots-duplicates/templates/action.cell.html

```diff
@@ -0,0 +1,15 @@
+<div class="ui-grid-cell-contents text-right" uib-dropdown dropdown-append-to-body uib-dropdown-toggle ng-if="!row.groupHeader">
+  <a href>
+    <span data-method="action" translate>FORM.BUTTONS.ACTIONS</span>
+    <span class="caret"></span>
+  </a>
+
+  <ul data-action="{{ rowRenderIndex}}"  class="dropdown-menu-right" bh-dropdown-menu-auto-dropup uib-dropdown-menu>
```

Comment:
You don't need the `{{rowRenderIndex}}`.  This is a testing anti-pattern (click in row by it's position in the grid).  Since there isn't an end to end test for this yet, you can just remove this binding.

Reply:
Removed